package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0-semantic-release-dev",
   "description": "The OPEN-RPC Specification",
   "main": "index.js",
+  "private": true,
```

Comment:
?

Reply:
this prevents package from being published

Reply:
![image](https://user-images.githubusercontent.com/1787231/71296389-491b0300-2334-11ea-9cf7-46e23730415f.png)
https://github.com/semantic-release/npm#options